paperpi/plugins/home_assistant/layout.py

```diff
@@ -0,0 +1,103 @@
+import os
+dir_path = os.path.dirname(os.path.realpath(__file__))
+
+four_sensors_and_media_player = {
+      'entity1': {
+          'type': 'TextBlock',
+          'image': False,
+          'max_lines': 1,
+          'width': 1/2,
+          'height': 0.2,
+          'padding': 20,
+          'font': dir_path+'/../../fonts/Anton/Anton-Regular.ttf',
+          'abs_coordinates': (0, 0),
+          'relative': False
+      },
```

Comment:
if you add `'mode': 'L'` to each block, it will allow HD gray-scale displays to anti-alias fonts and display gray-scale images. 1-Bit displays are automatically set to 1-bit mode, so this should not have any impact on other displays.
```
{
          'type': 'TextBlock',
          'mode': 'L',
          'image': False,
          'max_lines': 1,
          'width': 1/2,
          'height': 0.2,
          'padding': 20,
          'font': dir_path+'/../../fonts/Anton/Anton-Regular.ttf',
          'abs_coordinates': (0, 0),
          'relative': False
      }
```

Reply:
Thanks, I will add that. Now that I've seen how nice this looks I am kinda regretting not getting an HD display :D

Reply:
@logge the 9" HD is really, really nice. Because I'm dumb and break stuff, I ended up with two  and-a-half working 9" displays.

Reply:
I still have to mount mine in a frame so there is still plenty of opportunity to break it